<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>answerForm</class>
 <widget class="QWidget" name="answerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,4,0">
   <item row="1" column="0">
    <widget class="QLabel" name="incorrect_answer">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string>Incorrect Answer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="explanation">
     <property name="text">
      <string>Explanation</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::MarkdownText</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="answer">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string>Correct Answer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="continue_button">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
